orchestrate.py

```diff
@@ -306,27 +316,33 @@ def main():
     tornado.options.parse_command_line()
     opts = tornado.options.options
 
+    api_token = os.getenv('API_AUTH_TOKEN')
+    admin_token = os.getenv('ADMIN_AUTH_TOKEN')
+    proxy_token = os.environ['CONFIGPROXY_AUTH_TOKEN']
+    proxy_endpoint = os.environ.get('CONFIGPROXY_ENDPOINT', "http://127.0.0.1:8001")
+    docker_host = os.environ.get('DOCKER_HOST', 'unix://var/run/docker.sock')
+
     handlers = [
-        (r"/", LoadingHandler),
-        (r"/spawn/?(/user/\w+(?:/.*)?)?", SpawnHandler),
-        (r"/spawn/((?:notebooks|tree)(?:/.*)?)", SpawnHandler),
         (r"/api/spawn/?", APISpawnHandler),
-        (r"/(user/\w+)(?:/.*)?", LoadingHandler),
-        (r"/((?:notebooks|tree)(?:/.*)?)", LoadingHandler),
-        (r"/api/stats/?", APIStatsHandler),
-        (r"/stats/?", RedirectHandler, {"url": "/api/stats"}),
-        (r"/info/?", InfoHandler),
+        (r"/api/stats/?", APIStatsHandler)
```

Comment:
Ok, so if there's an API Token then we only have the /api/stats and /api/spawn endpoints. Seems alright.